src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAHelloWorldProject.cs

```diff
@@ -1072,6 +1072,61 @@ public void It_publishes_with_full_path_publish_profile()
                 .Pass();
         }
 
+        [Theory]
+        [InlineData("invalidProfile", true)]
+        [InlineData("invalidProfile.pubxml", true)]
+        [InlineData("..\\Properties\\PublishProfiles\\invalidProfile.pubxml", true)]
+        [InlineData("testProfile", false)]
+        [InlineData("testProfile.pubxml", false)]
+        [InlineData("..\\Properties\\PublishProfiles\\testProfile.pubxml", false)]
+        [InlineData("", false)]
+        public void It_warns_with_an_invalid_publish_profile(string publishProfile, bool shouldWarn)
+        {
+            var tfm = ToolsetInfo.CurrentTargetFramework;
+
+            var testProject = new TestProject()
+            {
+                Name = "ConsoleWithPublishProfile",
+                TargetFrameworks = tfm,
+                ProjectSdk = "Microsoft.NET.Sdk;Microsoft.NET.Sdk.Publish",
```

Comment:
Is it the normal customer scenario here that the project would specify the Publish Sdk?

Reply:
For the console apps, the publish SDK is not required. But we do add the publish sdk for console projects under specific circumstances like publishing the consoleapp as a WebJob to Azure App service etc... 

In this case the behavior should be same with or w/o the sdk.. I will add a test for that scenario as well.